FutureFeatures.md

```diff
@@ -273,14 +273,19 @@ quadruple precision.
 
 ## Floating point library intrinsics
 
-These operations aren't needed because they can be implemented in WebAssembly
-code and linked into WebAssembly modules as at small size cost, and this avoids
-a non-trivial specification burden of their semantics/precision. Adding these
-intrinsics would allow for better high-level backend optimization of these
-intrinsics that require builtin knowledge of their semantics. On the other
-hand, a code generator may continue to statically link in its own
-implementation since this provides greater control over precision/performance
-tradeoffs.
+These operations aren't needed for the MVP because they can be implemented in
+WebAssembly code and linked into WebAssembly modules at small size cost (as is
+traditionally done through C libraries such as libm). This avoids a non-trivial
+specification burden for their semantics, precision, and performance.
```

Comment:
Yes to the "race to the bottom" argument, but it's encompassed by having a specification for precision so I don't think it's worth mentioning (not everyone knows about transcendental cache sins, let's not dive into too many details).

I'm not sure getting consistent performance across implementations is really a goal. Code is code! If implementations lag behind then they need to work harder.